utils/node-loader/src/utils.rs

```diff
@@ -229,16 +230,30 @@ pub fn get_wasm_gen_config(
         .into_iter(),
     );
 
-    let mut params_config = SyscallsParamsConfig::default();
-    params_config.add_rule(ParamType::Regular(RegularParamType::Alloc), (1..=10).into());
-    params_config.add_rule(
-        ParamType::Regular(RegularParamType::Free),
+    let mut params_config = SyscallsParamsConfig::default_regular();
+    params_config.set_rule(RegularParamType::Alloc, (1..=10).into());
+    params_config.set_rule(
+        RegularParamType::Free,
         (initial_pages..=initial_pages + 50).into(),
     );
+    params_config.set_ptr_rule(PtrParamAllowedValues::Value(0..=0));
+
+    let syscall_destination = NonEmpty::collect(
+        existing_programs
+            .filter(|&pid| pid != ProgramId::default())
+            .map(|pid| pid.into()),
+    )
+    .map(SyscallDestination::ExistingAddresses)
+    .unwrap_or(SyscallDestination::Source);
+
+    params_config.set_ptr_rule(PtrParamAllowedValues::ActorId(syscall_destination.clone()));
+    params_config.set_ptr_rule(PtrParamAllowedValues::ActorIdWithValue {
+        actor: syscall_destination.clone(),
+        range: 0..=0,
```

Comment:
node loader does not support value fuzzing yet?

Reply:
well, yeah, we can remove that completely actually, as node-loader is dead until we finish up things with `runtime-fuzzer`.